File: 2.0/java/calls/getCallRecordingMp3ByName.java
import com.callfire.api.client.CallfireClient;

import java.io.InputStream;

class ApiClientSample {
    public static void main(String[] args) {
        CallfireClient client = new CallfireClient("api_login", "api_password");
        InputStream is = client.callsApi().getCallRecordingMp3ByName(1234L, "testName");
    }
}

File: 2.0/csharp/calls/getCallRecordingMp3ByName.cs
using System.IO;
using CallfireApiClient;

public class ApiClientSample
{
    public static void Main(string[] args)
    {
        var client = new CallfireClient("api_login", "api_password");
        Stream stream = client.CallsApi.GetCallRecordingMp3ByName(1234, "testName");
    }
}
